### DISCUSSION TITLE
how do I set the size of plot when using httpgd as plot device

### DISCUSSION THREAD

**Discussion by TTTPOB** [REditorSupport/vscode-R#978] [Q&A]
when using png file as the plot device I can use `vsc.dev.args` option to set the width, height and the resolution of the plot.
But when I am using `httpgd` device as plot backend, I don't know where to set these options (the default plot size is too large for me, I need to click + to zoom in multiple times)

**Comment by renkun-ken** [comment]
@nx10 Does it make sense to use R options to control the default zoom level?

**Reply by nx10** [reply]
I think this should be handled client side as it depends on the monitor the plot is viewed on. I am not sure how vscode handles screen DPI, but maybe the default zoom level should follow vscode UI scaling.
